routing/maxspeed_serialization.hpp

```diff
@@ -0,0 +1,5 @@
+#pragma once
```

Comment:
Why do we need this?

Reply:
It'll filled in next commit(s) in this PR.

Reply:
This idea is to place the code for serialization and deserialization of maxspeed section to routing lib and then use it.